src/pages/Groups/Group2/Screen2.tsx

```diff
@@ -1,7 +1,77 @@
 import React from 'react'
+import {
+  MainContainer,
+  Screen,
+  Picture,
+  BookingTitle,
+  Title,
+  CCA,
```

Comment:
namings such as "CCA", "Title", "Pen" are mostly not descriptive.

Try to think of explaining / describing the object to a teammate that hasn't seen this component. 
`screen2PageTitle`
`screen2PostTitle`
`screen2EventTitle`
all may be misunderstood as a "title"

Same goes for `Confirm` vs `ConfirmButton`

Reply:
Fixed in 7ef24b0